src/PhpSpec/Console/Application.php

```diff
@@ -39,12 +41,12 @@ class Application extends BaseApplication
      */
     public function __construct($version)
     {
-        $this->container = new ServiceContainer();
+        $this->container = new IndexedServiceContainer();
         parent::__construct('phpspec', $version);
     }
 
     /**
-     * @return ServiceContainer
+     * @return IndexedServiceContainer
```

Comment:
Yep, good catch - I'm considering removing this BTW, it's currently only used for testing? It should be `@internal` anyhow